api/identifiers/serializers.py

```diff
@@ -45,6 +55,33 @@ def self_url(self, obj):
             },
         )
 
+    def create(self, validated_data):
+        node = self.context['view'].get_node()
```

Comment:
Not allowing DOI's for withdrawn registrations, private nodes, and requiring admin perms to mint are enforced on permission classes.